README.md

```diff
@@ -84,6 +84,10 @@ class User extends Model {
 }
 ```
 
+#### `defaultAttributes: ObservableMap`
+
+An `ObservableMap` that holds the default attributes of the model. {} by default.
```

Comment:
why does it need to be observable? does it change?

Reply:
w00ps wrong documentation, it is not an observable.